docs/rules/no_sync_fn_in_async_fn.md

```diff
@@ -0,0 +1,37 @@
+Disallow sync function inside async function
+
+Using sync functions like `Deno.readTextFileSync` blocks the deno event loop so
+its not recommended to use it inside of an async function, because its stops
+progress of all other async tasks.
```

Comment:
```suggestion
Using sync functions like `Deno.readTextFileSync` blocks the deno event loop so
it's not recommended to use it inside of an async function, because it stops
progress of all other async tasks.
```